package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "wos",
+  "version": "1.0.0",
+  "description": "Wall of Sayings",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon app.js",
```

Comment:
Maybe rename this to `start-dev` or `start:dev` or `start:nodemon` since you won't use nodemon in production